hw/ip/otp_ctrl/data/otp_ctrl_testplan.hjson

```diff
@@ -229,7 +229,7 @@
             As mentioned in `prim_count_check`, `prim_fsm_check` and `prim_double_lfsr_check`.
 
             Checks:
-            - TODO, add additional checks if applicable.
+            - Check the value of status register according to where the fault is injected.
```

Comment:
If I remember correctly, you checked some OTP behavior or state after fault injection. Can you also add that?

Reply:
Oh yes, that is right. i see. Let me add that as well. thanks for catching that.